src/scripts/main.js

```diff
@@ -1,3 +1,71 @@
 'use strict';
 
 // write code here
+const table = document.querySelector('tbody');
+const buttonAddRow = document.querySelector('.append-row.button');
+const buttonAddColumn = document.querySelector('.append-column.button');
+const buttonRemoveColumn = document.querySelector('.remove-column.button');
+const buttonRemoveRow = document.querySelector('.remove-row.button');
+
+document.addEventListener('click', () => {
+  const columns = document.querySelector('tr').children.length;
+  const rows = document.querySelectorAll('tr').length;
```

Comment:
Similar to the previous comment, ensure that there are 'tr' elements present before counting them to avoid potential errors.